src/DatePicker/CalendarActionButtons.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import FlatButton from '../FlatButton';
+
+class CalendarActionButton extends React.Component {
+  static propTypes = {
+    cancelLabel: React.PropTypes.node,
+    okLabel: React.PropTypes.node,
+    onTouchTapCancel: React.PropTypes.func,
+    onTouchTapOk: React.PropTypes.func,
+    wordings: React.PropTypes.object,
```

Comment:
Why multiple props for the same purpose? Single source of truth should rule here